Sources/Controllers/2.0/BottomSheet/SettingsActionBottomSheet/OACopyProfileBottomSheetViewController.h

```diff
@@ -0,0 +1,43 @@
+//
+//  OACopyProfileBottomSheetViewController.h
+//  OsmAnd Maps
+//
+//  Created by Anna Bibyk on 05.08.2020.
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "OABottomSheetViewController.h"
+#import "OABottomSheetTwoButtonsViewController.h"
+#import "OAApplicationMode.h"
+
+@protocol OACopyProfileBottomSheetDelegate <NSObject>
+
+@required
+
+- (void) onCopyProfileCompleted;
+- (void) onCopyProfileDismessed;
+
+@end
+
+@interface OACopyProfileBottomSheetViewController : UIView<UITableViewDelegate, UITableViewDataSource>
```

Comment:
A class that inherits from UIView cannot be named ViewController. It is very confusing.

Reply:
Also, it would be better to extract the drag behavior to a base class, so we could re-use it in the future.